features/FxA-57-verify-reminder/README.md

```diff
@@ -12,7 +12,17 @@ As a new Firefox Accounts user,
  The first email arrives after 48 hours since sign up.
  
  The second email arrives after 168 hours since sign up.
- 
+
+## Success criteria
+
+We are planning to get the results from the following metrics:
+
+- Measure number of people that verify using the first email and second email.
+We would use the total number of emails sent vs verified.
+
+- The ultimate measure of success for this feature is if the user connects to sync.
+We will use the retention metrics pipeline to measure "account.signed" events that happened after "verification reminder" events
```

Comment:
Does the "verification reminder" event indicate that a reminder was sent, or that it was acted on?

We could be more explicit here as well, with something like "the percentage of 'verification reminder sent' events that are followed by an 'account.signed' event up to X days later".